canonical/snapcraft.io
# chore: Refactor publisher app

## Done
Refactored validation sets into the publisher app

## How to QA
- Go to https://snapcraft-io-4842.demos.haus/ and login
- Click your name in the header and check that "My validation sets" is there
- Go to https://snapcraft-io-4842.demos.haus/validation-sets
- Check that the page loads

## Testing
- [x] No testing required (explain why): Mostly just moving files
